### PR TITLE
Move test explorer to Test view container

### PR DESCRIPTION
**Pull request by sandy081**
Hi,

I am developer from VS Code and recently we added a new test view container. Refer to the release notes [here](https://github.com/Microsoft/vscode-docs/blob/vnext/release-notes/v1_23.md#contributions-to-the-activity-bar). 

I am going through the popular extensions in the market place those contribute test related custom views and moving those view to Test view container.

I would request to accept this PR and publish the extension with changes.

Thanks

### CODE REVIEW

File: extension/package.json [hunk 1]
```diff
@@ -42,7 +42,7 @@
             "./server/com.microsoft.java.test.plugin-0.6.0.jar"
         ],
         "views": {
-            "explorer": [
+            "test": [
```
Comment (yaohaizh) [inline extension/package.json]: Do we need update the VSCode engine version for this?

Comment (sandy081) [reply extension/package.json]: @yaohaizh Yes good point. Will update that.

**Comment by ansyral** [comment]
@sandy081 thanks for your contribution, could you help update https://github.com/Microsoft/vscode-java-test/blob/master/.travis.yml#L11?

**Comment by sandy081** [comment]
@ansyral Does the value needs to be set to 1.23?

**Comment by ansyral** [comment]
@sandy081  yes, we have to update it, otherwise the test won't pass.

**Comment by sandy081** [comment]
@ansyral I was on vacation and hence could not get back on to this.

Created another PR for the travis change -
https://github.com/Microsoft/vscode-java-test/pull/196

**Pull request merged**